core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -170,9 +170,7 @@ trait OneAndLowPriority2 extends OneAndLowPriority1 {
   implicit def oneAndTraverse[F[_]](implicit F: Traverse[F]): Traverse[OneAnd[F, ?]] =
     new Traverse[OneAnd[F, ?]] {
       def traverse[G[_], A, B](fa: OneAnd[F, A])(f: (A) => G[B])(implicit G: Applicative[G]): G[OneAnd[F, B]] = {
-        val tail = F.traverse(fa.tail)(f)
-        val head = f(fa.head)
-        G.ap2[B, F[B], OneAnd[F, B]](G.pure(OneAnd(_, _)))(head, tail)
+        G.map2Eval(f(fa.head), Later(F.traverse(fa.tail)(f)))(OneAnd(_, _)).value
```

Comment:
Same thing here, since we're immediately calling `.value` we can just use `Always`.